StreetHawk/Classes/Core/Publish/SHApp.m

```diff
@@ -1905,6 +1917,7 @@ -(void)registerOrUpdateInstallWithHandler:(SHCallbackHandler)handler
 -(BOOL)checkInstallChangeForLaunch
 {
     NSString *sentAppKey = [[NSUserDefaults standardUserDefaults] objectForKey:SentInstall_AppKey];
+    NSString *sentSegmentId = [[NSUserDefaults standardUserDefaults] objectForKey:SentInstall_SegmentId];
```

Comment:
In later code it should check whether this change, and do something.

Reply:
So, if segment id won't be changed in the lifecycle of an install, we don't need to put it here right? I will delete this line